src/color.class.js

```diff
@@ -185,6 +185,29 @@
     },
 
     /**
+     * Returns color representation in HEXA format
+     * @return {String} ex: FF5555CC
+     */
+    toHexa: function() {
+      var source = this.getSource(), r, g, b, a;
+
+      r = source[0].toString(16);
+      r = (r.length === 1) ? ('0' + r) : r;
+
+      g = source[1].toString(16);
+      g = (g.length === 1) ? ('0' + g) : g;
+
+      b = source[2].toString(16);
+      b = (b.length === 1) ? ('0' + b) : b;
+
+      a = source[3] * 255;
+      a = a.toString(16);
+      a = (a.length === 1) ? ('0' + a) : a;
+
+      return r.toUpperCase() + g.toUpperCase() + b.toUpperCase() + a.toUpperCase();
```

Comment:
all this could be `return this.toHex() + a.toUpperCase()` so we do not need to duplicate toHex code again.